File: web/components/collapse/collapse-demo.ts
/// <reference path="../../../tsd.d.ts" />
import {Component, View} from 'angular2/angular2';
import {Collapse} from '../../../lib/index';

// webpack html imports
let template = require('./collapse-demo.html');

@Component({
  selector: 'collapse-demo'
})
@View({
  template: template,
  directives: [Collapse]
})
export class CollapseDemo {
  public isCollapsed:boolean = false;
}

File: web/components/collapse/collapse-demo.dart
import "package:angular2/angular2.dart";
import 'package:ng2-strap/collapse/collapse.dart';


@Component(
    selector: "collapse-demo",
    templateUrl: 'collapse-demo.html',
    directives: const [Collapse])
class CollapseDemo {
  bool isCollapsed = false;
}
